product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml

```diff
@@ -0,0 +1,281 @@
+product: EDB Postgres Distributed
+version: 5.6.0
+date: 15 October 2024
+intro: |
+ EDB Postgres Distributed 5.6.0 includes a number of enhancements and bug fixes.
+ Please add more text here.
+highlights: |
+ - Streaming Transaction support with SDW.
+ - GROUP COMMIT and SYNCHRONOUS COMMIT support DEGRADE ON.
```

Comment:
Might merge the, the next one (origin group) and the improvement in the sync commit behavior around deadlocks with writer (that I also commented on) into one big highlight talking about improvements in commit scopes maybe?

Reply:
Also if we count the new usage stats (bdr.stat_commit_scope), and current info about backends using them (bdr.stat_commit_scope_state)

Reply:
And the ability to use parent group's commit scope (which is what makes ORIGIN_GROUP appealing)

Reply:
I think thats everything covered.